### PR TITLE
Add http health status endpoint for container liveness probes 

### PR DESCRIPTION
**Pull request by divyagandhisethi**
### Issue describing the changes in this PR
Adding health check endpoint for container liveness probe. Applies only to LinuxConsumption
resolves #issue_for_this_pr

### Pull request checklist

* [x] My changes **do not** require documentation changes
    * [ ] Otherwise: Documentation issue linked to PR
* [ ] My changes **should not** be added to the release notes for the next release
    * [x] Otherwise: I've added my notes to `release_notes.md`
* [ ] My changes **do not** need to be backported to a previous version
    * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [x] I have added all required tests (Unit tests, E2E tests)

### Additional information

Additional PR information

### CODE REVIEW

**Review by balag0 — DISMISSED** [review]

File: src/WebJobs.Script.WebHost/.config/dotnet-tools.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "version": 1,
```
Comment (fabiocav) [inline src/WebJobs.Script.WebHost/.config/dotnet-tools.json]: Is this intentional?

Comment (divyagandhisethi) [reply src/WebJobs.Script.WebHost/.config/dotnet-tools.json]: No. removed now.

**Comment by divyagandhisethi** [comment]
@fabiocav can you please approve again? I have addressed your comment above.

**Comment by fabiocav** [comment]
@divyagandhii is the existing status API not sufficient for what you want to do?

**Comment by divyagandhisethi** [comment]
> 
> 
> @divyagandhii is the existing status API not sufficient for what you want to do?

@fabiocav Ideally it would have worked. However there are subtle differences between the callers of the 2 endpoints. This new endpoint will only be called by the Atlas team's container liveness probe feature to monitor http health. So for example we cannot have this endpoint authenticated since we cannot control the request. 

The admin/host/status endpoint is called by Antares I believe and is an authenticated call. 

So I would prefer to have 2 different endpoints and narrow down the scope of this new endpoint.

**Comment by fabiocav** [comment]
The status API is called by a variety of clients, not just Antares.

We should NOT be adding an unauthenticated `admin` API. In addition to runtime checks, there are also infrastructure checks made when a request hits one of those endpoints.

If you just need an unauthenticated call, would a call to the root be sufficient? Is there a reason why the health check can't authenticate the call? Again, we don't want to introduce an open endpoint, particularly under that segment, into all functions.

**Comment by divyagandhisethi** [comment]
> 
> 
> The status API is called by a variety of clients, not just Antares.
> 
> We should NOT be adding an unauthenticated `admin` API. In addition to runtime checks, there are also infrastructure checks made when a request hits one of those endpoints.
> 
> If you just need an unauthenticated call, would a call to the root be sufficient? Is there a reason why the health check can't authenticate the call? Again, we don't want to introduce an open endpoint, particularly under that segment, into all functions.

Calls to root are authenticated if the host is specialized. In Linux Consumption, the only way to authenticate is through the request headers since the workers are outside of Antares.  Liveness probe doesn't support adding custom request headers.

I will abandon this PR.

**Pull request closed without merging**